src/constants/chains.ts

```diff
@@ -18,6 +18,20 @@ export enum SupportedChainId {
   POLYGON_MUMBAI = 80001,
 }
 
+export const CHAIN_IDS_TO_NAMES = {
```

Comment:
have you considered just putting that in `chain_ids` itself?

Reply:
do you mean in `ALL_SUPPORTED_CHAIN_IDS`? i think its fine to separate it now